test/stages/stage1-vpc-subnet.tf

```diff
@@ -10,7 +10,7 @@ module "dev_pub_subnet" {
   subnet_cidrs       = var.pub_subnet_cidrs
   availability_zones = var.availability_zones
   gateways           = [module.dev_igw.igw_id]
-  acl_rules          = var.acl_rules_pub
+  #acl_rules          = var.acl_rules_pub
```

Comment:
Same here... if not needed, can we get rid of these commented sections?

Reply:
Kept it as placeholder for testing with custom acls